test/helpers/helpers.js

```diff
@@ -1,5 +1,6 @@
 const marked = require('../../src/marked.js');
 const htmlDiffer = require('./html-differ.js');
+const assert = require('assert').strict;
```

Comment:
```suggestion
const assert = require('assert');
```

`.strict` is unnecessary since we only use `strictEqual`

Reply:
done